components/Workspaces/TrackedReposTable.tsx

```diff
@@ -0,0 +1,93 @@
+import { BiBarChartAlt2 } from "react-icons/bi";
+import { FaPlus } from "react-icons/fa6";
+import { FaTrashAlt } from "react-icons/fa";
+import Button from "components/atoms/Button/button";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "components/shared/Table";
+import { Avatar } from "components/atoms/Avatar/avatar-hover-card";
+
+interface TrackedReposTableProps {
+  repositories: {
+    owner: string;
+    name: string;
+  }[];
+  onAddRepos: () => void;
+  onRemoveTrackedRepo: () => void;
+}
+
+const EmptyState = ({ onAddRepos }: { onAddRepos: () => void }) => {
+  return (
+    <div className="grid place-content-center gap-4 my-8">
+      <BiBarChartAlt2 className="border rounded p-1 w-12 h-12 mx-auto" />
+      <div className="grid w-max max-w-sm mx-auto">
+        <span className="text-center text-lg font-semibold mb-2">Add repositories to track</span>
+        <p className="text-sm text-center">
+          Search and select the repositories you want to track and get insights on your entire Github ecosystem
+        </p>
+      </div>
+      <Button variant="primary" className="w-max mx-auto" onClick={onAddRepos}>
+        <FaPlus className="mr-2 text-lg" />
+        Add repositories
+      </Button>
+    </div>
+  );
+};
+
+export const TrackedReposTable = ({ repositories, onAddRepos, onRemoveTrackedRepo }: TrackedReposTableProps) => {
+  return (
+    <div className="grid gap-4">
+      <div>
+        <h2 className="flex gap-1 semi-bold text-lg mb-2">
+          Repositories Tracked<span className="text-red-900">*</span>
+        </h2>
+        <p className="text-sm">Select the organizations or repositories you want to track</p>
+      </div>
+      <div className="border border-light-slate-7 rounded">
```

Comment:
I wasn't positive about the colours in some of the stuff in this component @isabensusan as the Tailwind classes don't exist in our Figma files. We can tweak these during our sync tomorrow.